src/browser/iframe.js

```diff
@@ -23,14 +68,14 @@
     }),
     isVisible: VirtualAttribute((node, current, past) => {
       if (current != past) {
-        if (node.setVisible) {
+        if (node.setVisible && !node.isSetVisibleBroken) {
```

Comment:
execute `setVisible` even if `isSetVisibleBroken` is `true`, an catch the exception. When you catch the exception, if `isSetVisibleBroken` just keep going, if `isSetVisibleBroken` is false, throw the exception again.

Something along these lines:

```
if (node.setVisible) {
  try {
    node.setVisible()
  } catch(e) {
    if (!expectIsSetVisibleBroken) {
       throw(e);
    }
  }
}
```

Just in case we magically fix `setVisible` in Gecko and we forget to remove this hack.